### ISSUE TITLE
JATS Reader: ignore article/processing-meta/restricted-by element

### ISSUE THREAD

**Issue by castedo** [jgm/pandoc#8438]
With pandoc 2.19.2 and 
[iyab229.xml.txt](https://github.com/jgm/pandoc/files/9995838/iyab229.xml.txt)
```
pandoc iyab229.xml --from jats --to html --mathjax -s -o test.html
```
This XML is straight from what is stored in PubMed Central.

GOT:
[test.html.txt](https://github.com/jgm/pandoc/files/9995840/test.html.txt)
Under the abstract and before the intro there is the text "pmc".

EXPECTED:
The text "pmc" not to appear anywhere.

**Comment by kamoe** [comment]
Since `<restricted-by>` is metadata, an approach could be to add a case for "processing-meta" in the `parseBlock` function, along the lines of 

`"processing-meta" -> parseMetadata e`  

plus an additional getter in the `parseMetadata` function, to capture the relevant content as metadata without displaying it on the output.

**Comment by castedo** [comment]
I can not think of a big downside to the above approach. It definitely is better than returning the contents as regular text in the body of a document.

That said, just skipping and ignoring the entire `<processing-meta>` element seems equally good, and much less work.

If somebody wants to pull data out of `<processing-meta>` I wager they will be better served using a real XML parser rather pandoc to extract that data. Pandoc is the right tool to extract generic universal document data out of JATS XML to be converted to another (non-JATS) document format. But for data highly specific to JATS XML, a real XML parser makes more sense than pandoc.

**Comment by jgm** [comment]
I'd actually be fine with just skipping the processing-meta element entirely.  This would be easy to implement and is a definite improvement over what we have now.

**Comment by castedo** [comment]
Out of curiosity I tested what happens with another XML application that pandoc parses: XHTML.

Pandoc returns
```
html child
Yep, I am text.
```
for the following XHTML
```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<processing-meta>
  <restricted-by>html child</restricted-by>
</processing-meta>
<head>
  <processing-meta>
    <restricted-by>head child</restricted-by>
  </processing-meta>
  <title>XHTML Transitional Template</title>
  <unknown>NOT_TEXT</unknown>
</head>
<body>
<ignore>Yep, </ignore>I am text.
</body>
</html>
```

I think a good model for how JATS `<article><processing-meta>` should be handled is to be the way pandoc handles
`<html><head><processing-meta>`, and not the way `<html><processing-meta>` is handled.

Perhaps one way to reconcile this is that HTML is "by default" a container of marked up text with some elements, like `<html><head>` a container of data. In contrast JATS XML is defacto a container of data by default with specific elements like `<article><body>` as marked up text.

**Comment by kamoe** [comment]
@jgm Simplest fix by ignoring the <processing-meta> element altogether is here https://github.com/jgm/pandoc/pull/9057

@castedo I will happily add some more sophistication to this solution if we find this element needs any treatment at all, further down the line. For now it seems all we need to do is ignore it.

**Comment by castedo** [comment]
Good plan, thanks!

**Comment by jgm** [comment]
Closed by #9057 (I think).
